interfaces/IBF-dashboard/src/app/components/map/map.component.ts

```diff
@@ -533,48 +534,88 @@ export class MapComponent implements OnDestroy {
   private createMarkerStationPopup(markerProperties: Station): string {
     const percentageTrigger =
       markerProperties.fc / markerProperties.trigger_level;
-    const color = percentageTrigger < 1 ? '#a5d4a1' : '#d7301f';
-
-    const fullWidth = 50;
+    const color = percentageTrigger < 1 ? 'lightgrey' : 'salmon';
```

Comment:
These properties will have to be based on Glofas probability (add it to the endpoint) instead of this percentageTriger.